ocs_ci/deployment/hosted_cluster.py

```diff
@@ -1824,13 +1824,33 @@ def hypershift_cluster_factory(
                 ] = running_ocp_version
 
             cluster_path = create_cluster_dir(cluster_name)
-            kubeconf_path = (
+            logger.info("Cluster path: %s", cluster_path)
+            def_client_config_dict["ENV_DATA"]["cluster_path"] = cluster_path
+
+            kubeconf_paths = (
                 hosted_clients_obj.download_hosted_clusters_kubeconfig_files(
                     {cluster_name: cluster_path}
                 )
             )
+            if not len(kubeconf_paths):
+                from ocs_ci.framework.exceptions import ClusterKubeconfigNotFoundError
```

Comment:
Could be this import done in the beginning of the file?

Reply:
fixed in https://github.com/red-hat-storage/ocs-ci/pull/12041